sql/updates/post-update.sql

```diff
@@ -18,10 +23,19 @@ BEGIN
         -- this case.
         IF ts_version < '2.0.0' THEN
             EXECUTE format('ALTER VIEW %s SET (timescaledb.materialized_only=%L) ', vname::text, materialized_only);
-        ELSE
+        ELSIF ts_version < '2.7.0' THEN
             EXECUTE format('ALTER MATERIALIZED VIEW %s SET (timescaledb.materialized_only=%L) ', vname::text, materialized_only);
+        ELSE
+            SET log_error_verbosity TO VERBOSE;
+            RAISE WARNING 'find the name of CAGG before the call %', vname::text;
+            CALL @extschema@.post_update_cagg_try_repair(vname);
+            RAISE WARNING 'find the name of CAGG after the call %', vname::text;
         END IF;
```

Comment:
Not sure if this logic is correct because it means that if I'm upgrading to 2.7.0 we'll not execute the `ALTER MATERIALIZED VIEW` anymore... and as far I know it's necessary for users upgrading from versions before 2.0.0

Reply:
`cagg_rebuild_view_definition()` will do the same thing as `ALTER MATERIALIZED VIEW` for any older CAGG views. It will skip any view without partials and it will print warnings on buggy views.